packages/react/src/components/DataTable/stories/expansion/DataTable-expansion-story.scss

```diff
@@ -1,10 +1,6 @@
 @import '~@carbon/type/scss/type';
 @import '~@carbon/themes/scss/themes';
 
-.demo-expanded-td td {
-  padding-left: 4rem;
-}
-
 .demo-inner-container-header {
```

Comment:
Do we want to just add this to the expandable styles?

Reply:
Should already be getting it from this selector 👍🏻 

`tr.bx--parent-row.bx--expandable-row+tr[data-child-row] td`

<img width="866" alt="Screen Shot 2021-04-12 at 9 12 50 AM" src="https://user-images.githubusercontent.com/11928039/114426785-436faf00-9b6f-11eb-9ec9-7859bac3df7d.png">